app/components/toolbar.hbs

```diff
@@ -1,19 +1,20 @@
 <nav>
   <div class="au-c-app-chrome">
-    <AuToolbar @size="small" class="au-u-padding-bottom-none" as |Group|>
+    <AuToolbar @size="medium" @skin="tint" @border="bottom" as |Group|>
       <Group>
-        <AuLink @route="index" @skin="secondary" @icon="arrow-left">
-          {{t "navigation.backToFormOverview"}}
-        </AuLink>
-        <span class="au-c-app-chrome__entity">{{@model.label}}</span>
-      </Group>
-      <Group>
-        <ul class="au-c-list-horizontal au-u-padding-right-tiny">
+        <ul class="au-c-list-horizontal au-c-list-horizontal--small">
+          <li class="au-c-list-horizontal__item">
+            <AuLink
+              @route="index"
+              @skin="primary"
+              @icon="arrow-left"
+              @iconAlignment="left"
+            >
+              {{t "navigation.backToFormOverview"}}
+            </AuLink>
+          </li>
```

Comment:
Comes from

![image](https://github.com/lblod/frontend-form-builder/assets/36266973/065f7791-8f9a-4943-8411-377f4f5effbd)

To

![image](https://github.com/lblod/frontend-form-builder/assets/36266973/bd767dea-e41b-46ae-90bc-ff820f682550)

Appuniversum styling

![image](https://github.com/lblod/frontend-form-builder/assets/36266973/a133d860-c4e7-40eb-a9cd-1c6c7f418378)